Engine_Revit_UI/Convert/Structure/FromRevit/MaterialFragment.cs

```diff
@@ -155,9 +155,6 @@ private static IMaterialFragment Update(this IMaterialFragment materialFragment,
                 material.YoungsModulus = youngsModulus.X;
                 material.ThermalExpansionCoeff = thermalExpansionCoeff.X;
                 material.PoissonsRatio = poissonsRatio.X;
-                material.CustomData["Concrete Bending Reinforcement"] = structuralAsset.ConcreteBendingReinforcement;
-                material.CustomData["Concrete Shear Reinforcement"] = structuralAsset.ConcreteShearReinforcement;
-                material.CustomData["Concrete Shear Strength Reduction"] = structuralAsset.ConcreteShearStrengthReduction;
```

Comment:
This change affects @IsakNaslundBh heavily - worth him checking this just in case.

Reply:
Good point, I have deleted that somehow automatically. My gut feeling is that it had never been used anywhere as it is a mere `CustomData` key, not a property, but good to wait for @IsakNaslundBh's opinion on that.

Reply:
From the side of the core BHoM, removing this should be absolutely fine. No methods or similar rely on these custom data values being set (if we did, would have made it a core property of the concrete material).

Only risk I can think of is if someone has a project script set up that makes use of these parameters. I would expect that to be fairly unlikely, but can ofc not be 100% sure.

Just to check, why are they removed @pawelbaran ? Not saying it is wrong, but just to confirm.

Reply:
They are removed as the general notion set by @FraserGreenroyd is to avoid storing discipline-specific information in `CustomData`, assuming that if something is meant to be usable later on in the BHoM, it should have a property to be assigned to.

From now on, only parameter values are meant to be copied across to `CustomData` (they will ultimately be migrated to `Parameters` fragment).

Reply:
Does the above satisfy you @FraserGreenroyd @IsakNaslundBh?

Reply:
It does, but considering if they are worth keeping for now, as we do not have a strategy in place for how to handle them yet. The reinforcement parameters will never be (or never say never, but...) part of the Concrete material in structures, as we will want to handle reinforcement more explicitly, but, you might maybe still be able to pull the rest of the data (as revit generally works).

So would probably argue, lets keep, until another strategy for how to deal with those particular parameters is in place.

Reply:
After giving it another look I noticed this problem is likely to be much more convoluted - I have reverted the change in this PR and will rather challenge it with a wider slash in https://github.com/BHoM/Revit_Toolkit/issues/638. I believe this resolves the comment.